examples/jsm/controls/OrbitControls.js

```diff
@@ -292,17 +288,35 @@ class OrbitControls extends EventDispatcher {
 
 		}();
 
-		this.dispose = function () {
+		this.connect = function (domElement) {
```

Comment:
Just noticed that here are two missing spaces between the brackets and the parameter.

Reply:
Ah good catch! Not sure why my IDE didn't pick up the eslint errors